### PR TITLE
libjpeg-turbo: bump to 2.1.5.1

### PR DESCRIPTION
**Pull request by kenmays**
Built and unit tested on hrev58272 x86.

### CODE REVIEW

File: media-libs/libjpeg-turbo/licenses/IJG
Comment (jmairboeck) [inline media-libs/libjpeg-turbo/licenses/IJG:1]: The new content of this file is just an informational overview of the licenses and not the IJG license text itself. Please revert this. The licenses are listed correctly already in the recipe.

Comment (kenmays) [reply media-libs/libjpeg-turbo/licenses/IJG:1]: reverted... although the 'newer' one was here:[  ](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/README.ijg)

Comment (jmairboeck) [reply media-libs/libjpeg-turbo/licenses/IJG:1]: Yes, but that wasn't the text you had in this file.

Comment (kenmays) [reply media-libs/libjpeg-turbo/licenses/IJG:1]: Corrected. The site reflects a blanket 'license' that discusses and reflects on the three licenses in place and adds additional comments. I'll just use the newer IJG license in this case.

**Pull request merged**